src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,48 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+const showNoCommnets = () => (
```

Comment:
It looks like a React component, so it would be better to name it correspondingly - EmptyCommnets, NoComments, etc..
```suggestion
const EmptyCommnets = () => (
```